airbyte-integrations/connectors/destination-snowflake/src/test-integration/java/io/airbyte/integrations/destination/snowflake/SnowflakeInsertDestinationAcceptanceTest.java

```diff
@@ -41,6 +43,12 @@
 public class SnowflakeInsertDestinationAcceptanceTest extends DestinationAcceptanceTest {
 
   private static final NamingConventionTransformer NAME_TRANSFORMER = new SnowflakeSQLNameTransformer();
+  protected static final String NO_ACTIVE_WAREHOUSE_ERR_MSG = "No active warehouse selected in the current session. "
+      + " Select an active warehouse with the 'use warehouse' command.";
+
+  protected static final String NO_USER_PRIVILEGES_ERR_MSG = "Schema 'TEXT_SCHEMA' already exists, but current role "
```

Comment:
Suggestion to make this error message a text block so you don't need to use a string concatenation

https://github.com/airbytehq/airbyte/blob/9c2601013771ca0cd83b633303d40dcbc1a38b1a/airbyte-integrations/connectors/destination-redshift/src/main/java/io/airbyte/integrations/destination/redshift/operations/RedshiftSqlOperations.java#L72-L77

Reply:
Simplified expected message. If I make it as a text box - then it appears to be either very long or tests fail as it started considering new lines in actual msg also. Thanks